pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -43,14 +43,12 @@ import (
 )
 
 const (
-	// If the pod stays in unschedulableQ longer than the unschedulableQTimeInterval,
+	// If the pod stays in unschedulableQ longer than the podMaxUnschedulableDuration ,
```

Comment:
update the comment here to explain "DefaultPodMaxUnschedulableDuration" instead of "podMaxUnschedulableDuration".